core/commands/commit_compare.py

```diff
@@ -172,6 +172,11 @@ class GsCompareAgainstCommand(PanelActionMixin, WindowCommand, GitCommand):
     def run(self, target_commit=None, file_path=None, current_file=False):
         self._file_path = self.file_path if current_file else file_path
         self._target_commit = target_commit
+
+        # if there is a target, show option to compare against HEAD
+        if target_commit:
+            self.default_actions.insert(0, ["compare_against_head", "HEAD"])
```

Comment:
Ya, you are right.

PS: It seems that there are some random failures on CIs. I am tracking the issue.